### DISCUSSION TITLE
RSS Rules

### DISCUSSION THREAD

**Discussion by Str00pwafel** [VueTorrent/VueTorrent#486] [Q&A]
Hello,

I'm sorry if this is a basic noob question but how do I fill the rules for RSS feed?
If I'm looking for specific names or quality for example, and can I set it to a catergory automatically?

PS. amazing looking GUI, thanks for this work! :D

**Comment by WDaan** [comment]
Unfortunately the RSS feature isn't that complete yet, only the basics are supported for now.

**Reply by tor2dbear** [reply]
What is the unit for the "Feeds refresh interval"? Is it seconds or minutes?

Thanks!

**Reply by Larsluph** [reply]
It's in minutes, I may need to add a hint. Thanks for the feedback !

**Reply by Larsluph** [reply]
And it already has a hint 😅
![image](https://github.com/WDaan/VueTorrent/assets/22910497/5503043c-c17a-4666-8954-7a3d58ccfa71)

**Reply by tor2dbear** [reply]
Might need to update, missing for me :) Thanks!

**Reply by dan195794** [reply]
Can i see the Rss by feeds/folders ?
My All list is a mess and useless

**Reply by Larsluph** [reply]
@dan195794 You can click on each feed in the RSS to only display articles in that feed.
